NZZ.ch.js

```diff
@@ -1,224 +1,271 @@
-			});
+			ZU.processDocuments(articles, scrape);
 		});
 	} else {
-		scrape(doc, url);
+ 		scrape(doc, url);
 	}
 }
 
-/* Three types of articles: "Neue Zürcher Zeitung", "NZZ Online" and "NZZ am Sonntag" */
-function scrape(doc) {
-	//Zotero.debug("ibex scrape URL = " + doc.location.href);
-	var newItem = new Zotero.Item('newspaperArticle');
-	newItem.url = doc.location.href;
-	newItem.title = Zotero.Utilities.trimInternal(getXPath('//hgroup/h1', doc).textContent);
-	var date = ZU.xpathText(doc, '//hgroup/time/@datetime');
-	if (date) newItem.date = date.replace(/\d\d\:\d\d:\d\d/, "").trim();
-	newItem.publicationTitle = "Neue Zürcher Zeitung";
-	newItem.ISSN = "0376-6829";
-	newItem.language = "de";
-
-	var titleprefix = getXPath('//hgroup/h5', doc);
-	if ((titleprefix != null) && (Zotero.Utilities.trimInternal(titleprefix.textContent) != "")) {
-		newItem.shortTitle = newItem.title;
-		newItem.title = Zotero.Utilities.trimInternal(titleprefix.textContent) + ": " + newItem.title;
-	}
 
-	var subtitle = getXPath('//hgroup/h2', doc);
-	if ((subtitle != null) && (Zotero.Utilities.trimInternal(subtitle.textContent) != "")) {
-		newItem.shortTitle = newItem.title;
-		newItem.title += ": " + Zotero.Utilities.trimInternal(subtitle.textContent);
-	}
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	translator.setDocument(doc);
+	
+	translator.setHandler('itemDone', function (obj, item) {
 
-	var teaser = getXPath('//article/h5', doc);
-	if ((teaser != null) && (Zotero.Utilities.trimInternal(teaser.textContent) != "")) {
-		newItem.abstractNote = Zotero.Utilities.trimInternal(teaser.textContent);
-	}
-
-	var authorline = getXPath('//article/address/span', doc);
-	if (!authorline) authorline = getXPath('//h6//span[@class="author"]', doc)
-	if (authorline != null) {
-		authorline = Zotero.Utilities.trimInternal(authorline.textContent);
-		//assumption of authorline: "[Interview:|Von ]name1[, name2] [und Name3][, location]"
-		authorline = authorline.replace(/^Von /, "");
-		authorline = authorline.replace(/^Interview: /, "");
-		authorline = authorline.replace(/vor Ort /i, "");
-		//remove ", location"
-		authorline = Zotero.Utilities.trim(authorline.replace(/, \S*$/, ""));
-
-		var authors = authorline.split(/,|und/);
-		for (var i = 0; i < authors.length && authorline.length > 0; i++) {
-			newItem.creators.push(Zotero.Utilities.cleanAuthor(authors[i], "author"));
-			if (!newItem.creators[i].firstName){
-				newItem.creators[i].fieldMode = 1;
+		item.date = ZU.xpathText(doc, '//article[contains(@class, "content")]//time/@datetime');
+		
+		// Problem: also the place will be taken as part of the autor name
+		// e.g. <meta name="author" content="Matthias Müller, Peking">
+		// e.g. <meta name="author" content="Marco Metzler und Birgit Voigt" />
+		var authorString = ZU.xpathText(doc, '//meta[@name="author"]/@content');
```

Comment:
I don't know. It is already very hard to find a newspaper article with two authors. I haven't found any with more than two authors.